pkg/s3io/s3io.go

```diff
@@ -38,6 +39,7 @@ func parsePath(path string) (bucket, key string, err error) {
 	}
 	bucket = u.Host
 	key = u.Path
+	key = strings.TrimPrefix(key, "/")
```

Comment:
Nit: Shorten to `key = strings.TrimPrefix(u.Path, "/")` or `return u.Host, strings.TrimPrefix(u.Path, "/"), err`.